ui/web/index.html

```diff
@@ -3,10 +3,306 @@
+                        <option value="J">J</option>
+                        <option value="K">K</option>
+                        <option value="L">L</option>            
+                    </select>
+                </td>
+            </tr>
+            <tr>
+                <td>blockers
+                    <select name="blockers" id="blockers">
+                        <option value=""></option>
+                        <option value="A">A</option>
+                        <option value="B">B</option>
+                        <option value="C">C</option>
+                        <option value="D">D</option>
+                        <option value="E">E</option>
+                        <option value="F">F</option>
+                        <option value="G">G</option>
+                        <option value="H">H</option>
+                        <option value="I">I</option>
+                        <option value="J">J</option>
+                        <option value="K">K</option>
+                        <option value="L">L</option>            
+                    </select>
+                </td>
+                <td>max_estimate
+                    <select name="max_estimate" id="maxEstimate">
+                        <option value=""></option>
+                        <option value="A">A</option>
+                        <option value="B">B</option>
+                        <option value="C">C</option>
+                        <option value="D">D</option>
+                        <option value="E">E</option>
+                        <option value="F">F</option>
+                        <option value="G">G</option>
+                        <option value="H">H</option>
+                        <option value="I">I</option>
+                        <option value="J">J</option>
+                        <option value="K">K</option>
+                        <option value="L">L</option>            
+                    </select>
+                </td>
+                </td>
+            </tr>
+        </table>
+    </div> 
+
+    </fieldset>
+
+    <fieldset>
+
+        <legend>Load</legend>
+
+        <div id="place">
+
+            <input type="submit" name="but" id="loadTasksButton" value="Load">
+
+        </div>
+
+    </fieldset>
+    <fieldset>
+
+        <legend>Print</legend>
+
+            <table border="0" widtd="100%">
+                <tr>
+                    <td>Tasks
+                        <div id="choiceGraphic"></div>                      
+                    </td>
+                    <td>
+                        <input type="button" value="Print" id="printGraphicButton">
+                    </td>
+                </tr>
+                <tr>
+                    <td>
+                        <div id="chartdiv" style="height:500px;width:400px; "></div>
+                    </td>
+                </tr>
+                </table>
+    </fieldset>
+
+    <script> 
+            var applicationContext = {}; 
```

Comment:
It's better to have some object with base application constants, so let's create something like:

```
var applicationConstants = {
  LOAD_EXCEL_FILE_TO_SERVER_URL: 'http://localhost:5000/api/excelWorkbook',
  LOAD_TASKS_FROM_SERVER_ERL: 'http://localhost:5000/api/excelWorkbook/tasks',
  LOAD_NORMAL_DISTRIBUTION_FROM_SERVER_URL: 'http://localhost:5000/api/tasks/normalDistribution',
  NUMBER_OF_NORMAL_DISTRIBUTION_POINTS: 50
}

...
url: applicationConstants.LOAD_EXCEL_FILE_TO_SERVER_URL,
...
"numberOfPoints": applicationConstants.NUMBER_OF_NORMAL_DISTRIBUTION_POINTS
```

Reply:
Created object with base constants.